sentry/src/main/java/io/sentry/protocol/SdkVersion.java

```diff
@@ -240,8 +215,9 @@ public static final class Deserializer implements JsonDeserializer<SdkVersion> {
       }
 
       SdkVersion sdkVersion = new SdkVersion(name, version);
-      sdkVersion.packages = packages;
-      sdkVersion.integrations = integrations;
+      SentryIntegrationPackageStorage.getInstance().setPackages(packages);
```

Comment:
`h` this clears all packages and replaces them for the whole system. This will likely cause problems, e.g. when we deserialize a cached envelope on startup, we lose the current list.

Not sure how to solve yet.